pkgs/development/tools/electron/binary/generic.nix

```diff
@@ -86,7 +87,17 @@ let
   common = platform: {
     inherit pname version meta;
     src = fetcher version (get tags platform) (get hashes platform);
-    passthru.headers = headersFetcher version hashes.headers;
+    passthru =
+      let
+        headers-packed = headersFetcher version hashes.headers;
```

Comment:
Shall we use `fetchzip` in headersFetcher?

Reply:
Maybe, but the `update.py` script would also need to be change to handle unpacking.

Reply:
That's beyond me. The update script is to complex for me.

Reply:
Also then we would need to regen all sources which will take a while.